Tests/PINRemoteImageTests.m

```diff
@@ -204,6 +234,18 @@ - (NSURL *)imageFrom404URL
 - (void)didReceiveData:(NSData *)data forTask:(NSURLSessionTask *)task
 {
     self.task = task;
+    if (self.semaphore) {
```

Comment:
Can you add a comment saying which test this is for?

Reply:
Sure. 👍